kafka/resource_kafka_topic.go

```diff
@@ -260,7 +260,7 @@ func topicDeleteFunc(client *LazyClient, id string, t Topic) resource.StateRefre
 func topicRead(d *schema.ResourceData, meta interface{}) error {
 	name := d.Id()
 	client := meta.(*LazyClient)
-	topic, err := client.ReadTopic(name)
+	topic, err := client.ReadTopic(name, false)
```

Comment:
This is just reading existing cluster state, so don't refresh state, since it shouldn't have been changed by anything we've done.